File: test1.js
function reOrdering(text) {
    let splitted = text.split(' ');

    for (let i = 0; i < splitted.length; i++) {
        if (splitted[i][0] === splitted[i][0].toUpperCase()) {
            // Swap the current word with the first capitalized word (if not already at the first position)
            if (i > 0) {
                let temp = splitted[0];
                splitted[0] = splitted[i];
                splitted[i] = temp;
            }
        }
    }

    // Join the modified words back into a string using space as a separator
    let result = splitted.join(' ');
    console.log(result);
}

// Test cases
reOrdering('ming Yao');
reOrdering('Mano donowana');
reOrdering('wario LoBan hello');
reOrdering('bull color pig Patrick');

File: test1.py
def re_ordering(text):
    split_txt = text.split(' ')
    for i in range(len(split_txt)):
        if i == 0 or i == split_txt[0].capitalize():
            continue
        else:
            tmp = split_txt[0]
            split_txt[0] = split_txt[i]
            split_txt[i] = tmp
            print(' '.join(split_txt))

# Test cases
re_ordering('ming Yao')
re_ordering('Mano donowana')
re_ordering('wario LoBan hello')
re_ordering('bull color pig Patrick')
